Source/WebCore/dom/EventNames.json

```diff
@@ -1,5 +1,5 @@
 {
-    "DOMActivate": {"types": ["MouseClickRelated"]},
+    "DOMActivate": { "categories": ["MouseClickRelated"], "defaultEventHandler": true },
```

Comment:
```suggestion
    "DOMActivate": {
        "categories": ["MouseClickRelated"],
        "defaultEventHandler": true
     },
```
nit: Can we format like this?

Reply:
We can but I don't want each event to take up more than one line.